tdrs-frontend/src/assets/Modal.scss

```diff
@@ -1,13 +1,13 @@
 .modal {
-  position: fixed; 
-  z-index: 1; 
+  position: fixed;
+  z-index: 3;
```

Comment:
the actual change to fix the highlight effect when the modal was displayed. the rest are style changes made by my editor